modules/sharedaddy/sharedaddy.php

```diff
@@ -68,7 +68,14 @@ function sharing_email_send_post_content( $data ) {
 	$content  = sprintf( __( '%1$s (%2$s) thinks you may be interested in the following post:', 'jetpack' ), $data['name'], $data['source'] );
 	$content .= "\n\n";
 	$content .= $data['post']->post_title."\n";
-	$content .= get_permalink( $data['post']->ID )."\n";
+	
+	// Filtered share URL. Allows backwards compatibility.
+	if ( isset( $data['share_url'] ) && ! empty( $data['share_url'] ) ) {
```

Comment:
You don't need to check for both `isset()` and `! empty()` -- if it's not set, `empty()` will return true -- it can accept unset variables without causing issues.